proposals/NNNN-Dynamic-App-List-Sorting.md

```diff
@@ -74,12 +74,12 @@ Currently, there is no sequence provided that demonstrates how the icon is displ
 
 ## Potential downsides
 
-Since it is necessary to implement the large amount of modification due to implement multiple sorting methods, the developer must do the implementation carefully and also, it will take a long time.
+As there is a large modification required to implement multiple sorting methods, the developer must do the implementation carefully and it will take a long time. Also, it may need to remove the sorting function from the HMI.
 
 ## Impact on existing code
 
 The existing sorting process is left as it is. Furthermore, as processes/lists related to sorting are added, SDL Core is affected. Also, since the priority information of `appHMIType_sorting_list` is stored, the policy server is affected, too. However, I believe their changes are minor version changes because there are no changes in RPC and parameters.
 
 ## Alternatives considered
 
-The alternative sorting method is user defined sort order. This sorting method would be a feature that some OEMs would want. However, since this specific sorting method would only be part of the HMI and have no other SDL component integration, we think it is difficult for SDL component to realize this function. Therefore, we think it is better this function to include into the HMI Integration Guidelines and SDL Core Guides documentation.
\ No newline at end of file
+The alternative sorting method is user defined sort order. This sorting method would be a feature that some OEMs would want. However, since this specific sorting method would only be part of the HMI and have no other SDL component integration, we think it is difficult for SDL component to realize the user defined sort order function.
```

Comment:
```suggestion
The alternative sorting method is user-defined sort order. This sorting method would be a feature that some OEMs would want. However, since this specific sorting method would only be part of the HMI and have no other SDL component integration, we think it is difficult for SDL Core to contain the user-defined sort order function.
```
I've suggested rewording the last sentence based on your response [here](https://github.com/smartdevicelink/sdl_evolution/pull/1085#discussion_r508153699). Please commit this suggested change if I'm understanding correctly, or let me know if I misunderstood and change as you see fit.